tests/integration/primary_pool_tests.cpp

```diff
@@ -579,7 +579,7 @@ TYPED_TEST(PrimaryPoolTest, heuristic_75_percent_hwm)
   ASSERT_NO_THROW({ alloc.deallocate(a[2]); }); // 50% releasable
   ASSERT_EQ(pool->getBlocksInPool(), 4);
   ASSERT_NO_THROW({ alloc.deallocate(a[1]); }); // 75% releasable
-  ASSERT_EQ(pool->getBlocksInPool(), 2);        // Collapse happened
+  ASSERT_EQ(pool->getBlocksInPool(), 4);
```

Comment:
This happens because of the change from `>=` to `>` in the coalescing heuristic. Instead of coalescing at exactly 75% releasable it'll only coalesce when strictly more than 75% memory is free.

Reply:
This is also reverted with the latest changes.